libs/ai-endpoints/langchain_nvidia_ai_endpoints/_common.py

```diff
@@ -589,6 +610,13 @@ def get_available_models(
         **kwargs: Any,
     ) -> List[Model]:
         """Retrieve a list of available models."""
+
+        # set client for lora models in local-nim mode
+        if not self.is_hosted:
+            for model in self.client.available_models:
+                if model.model_type == "lora":
+                    model.client = filter
```

Comment:
instead of this, what if we avoid filtering when talking to a local nim?

Reply:
yeah, can do that